### ISSUE TITLE
Vertical text detection

### ISSUE THREAD

**Issue by carlosrochastoresace** [tesseract-ocr/tesseract#2928]
Hi all!
I've been struggling for a while with this problem.
I need to get text from an image column by column, by despite my all config efforts I only can read the text from the image line by line.
Can anyone help me please?
Cheers

**Comment by amitdo** [comment]
I don't know what you mean here.

Post an image and the command you used. Also post version information.

**Comment by vishal-kr-yadav** [comment]
Hi @amitdo 
i came across the same problem statement,
If the text in the image is written in horizontal direction then the library can easily extract.
But i have something in vertical direction it fails to identity
Example:
text in horizontal direction : **vishal123**
text in vertical direction :
v
i
s
h
a
l
1
2
3
Note : if i have only numbers in vertical direction then its able to extract the numbers. But with the alphabets it will fails or identify the wrong character

**Comment by NikhilShaw** [comment]
Hi @vishal-kr-yadav, did you come up with any solution?

**Comment by vishkry** [comment]
@NikhilShaw yes, I tried various way using OCR but it didnt give me that much accuracy. So i moved to object detection, considering each character as an object & then arrange based on the coordinate of each object

**Comment by NikhilShaw** [comment]
@vishkry Just curious, did you use any pre-trained model for object detection which works well for vertical text or did you annotate and trained your own model?
